shell/platform/darwin/ios/framework/Headers/FlutterEngine.h

```diff
@@ -99,10 +105,10 @@ FLUTTER_EXPORT
  * immediately.
  *
  * @param entrypoint The name of a top-level function from the same Dart
- *   library that contains the app's main() function.  If this is nil, it will
- *   default to `main()`.  If it is not the app's main() function, that function
- *   must be decorated with `@pragma(vm:entry-point)` to ensure the method is not
- *   tree-shaken by the Dart compiler.
+ *   library that contains the app's main() function.  If this is FlutterDefaultDartEntrypoint (or
+ *   nil) it will default to `main()`.  If it is not the app's main() function, that function must
```

Comment:
q: is it `main()`or `main`?  The Android embedding seems to use just `main` https://github.com/flutter/engine/blob/fde7c8c3340964c680da99106ea485fc9474ca29/shell/platform/android/io/flutter/embedding/android/FlutterFragment.java#L202

Reply:
I can confirm that the literal value on Android is just the name of the function, no `()`. For the purpose of documentation it might be useful to represent it as "main()" for clarity.